### DISCUSSION TITLE
Логика работы с долгами

### DISCUSSION THREAD

**Discussion by lewadedun** [ardov/zerro#127] [General]
Как я понимаю, в настоящее время дача в долг выглядит как расход за балансом? При этом если должник в данный месяц долг вернул, то программа это не учитывает, т.е. бюджет все равно меньше не сумму долга? Считаю, что это несовсем корректно. В случе возврата долга в бюджете этот расход учитываться не должен.

**Comment by glmars** [comment]
Если у категории (должника) поставить галку "Класть доходы в конверт", то дача и возврат долга друг друга "компенсируют". Попробуйте, то что надо получится?

**Reply by lewadedun** [reply]
Да... но почему теперь появилась сумма в "доступно"? По идее  - д.б. 0 вообще и строчка должна уйти...
![Снимок](https://user-images.githubusercontent.com/9382415/228937780-27e5bd00-dcd1-456b-9fee-9e9ad203653f.PNG)

**Reply by glmars** [reply]
В изначальном вопросе речь шла про дачу/возврат в одном месяце, или я неправильно понял? "Компенсация" произойдёт если в одном месяце это всё происходит.

**Reply by lewadedun** [reply]
Да, все в одном месяце: 08.03 долг, 11.03 возврат.

**Reply by glmars** [reply]
Галочка "Класть доходы в конверт" на конверте "Ащеулов" стоит? В операции дачи и возврата долга должник одинаково называется (буква в букву, заглавные, прописные)?

**Reply by lewadedun** [reply]
Конечно.

**Reply by glmars** [reply]
Понял. Это с прошлых месяцев перенеслось. Надо в прошлом месяце "Сбросить остаток" на этом конверте.

**Reply by lewadedun** [reply]
Эмм... Что-то не могу сообразить, как сбросить остаток...

**Reply by glmars** [reply]
Тыкнуть в первой колонке и выбрать в меню?

**Reply by lewadedun** [reply]
Вроде получилось. Спасибо ; )
